src/Websites/Websites.Test/ScenarioTests/WebAppTests.ps1

```diff
@@ -1106,6 +1107,7 @@ function Test-SetWebApp
 		# Set config properties
 		$webapp.SiteConfig.HttpLoggingEnabled = $true
 		$webapp.SiteConfig.RequestTracingEnabled = $true
+		$webapp.SiteConfig.FtpsState  = $ftpsState
```

Comment:
AllAllowed is the default value here so its already set by default on an app - the test should be to set or update this to a different value and validate that it works. and Add a test for the same on slots as well.